include/seqan3/std/ranges

```diff
@@ -172,7 +172,7 @@ using ::ranges::cpp20::views::all;
 template<viewable_range R>
 using all_t = ::ranges::cpp20::all_view<R>;
 } // namespace views
-// using ::ranges::cpp20::ref_view;
+using ::ranges::cpp20::ref_view;
```

Comment:
Does this also/Can we  disable the usage of `ranges::ref_view`?

Reply:
Not sure what you mean?

You mean that we forcefully disallow the usage of `ranges::ref_view`?

I don't know how, maybe you have an idea.